content/en/docs/setup/platform-setup/kops/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Kops
+description: Instructions to setup Kops for use with Istio.
+weight: 20
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/kops/
+    - /docs/setup/kubernetes/platform-setup/kops/
+keywords: [platform-setup,kubernetes,kops]
+---
+
+Follow these instructions to prepare an AWS cluster with Kops for Istio.
+
+When you install a new cluster with Kubernetes version 1.9+, the prerequisite to
```

Comment:
These instructions seems to be focusing on old versions of Kubernetes. Does that make sense given that Istio 1.5 only supports (tested on) 1.14, 1.15, and 1.16?